### PR TITLE
Closes #1631 - fix descriptions of singleline and multiline

### LINKED ISSUE #1631: about_Split - description of RegexMatch options 'Multiline' and 'Singleline' is incorrect
Issue Details
-----------------

In [/reference/6/Microsoft.PowerShell.Core/About/about_Split.md](https://github.com/PowerShell/PowerShell-Docs/blob/b9282305775d00f28ce71c6475e23408dccce7de/reference/6/Microsoft.PowerShell.Core/About/about_Split.md#L206
) the following description appears:

 - **Multiline**: Multiline mode recognizes the start and end of lines and strings. Valid only with RegexMatch. Singleline is the default.
 - **Singleline**: Singleline mode recognizes only the start and end of strings. Valid only with RegexMatch. Singleline is the default.

Referencing [PowerShell language issue 4712](https://github.com/PowerShell/PowerShell/issues/4712), this is incorrect because:

 - it suggests there is only one option and it is switched between 'single' or 'multi', but there are two options which can be independently switched on/off. Singleline is not the opposite of Multiline.
 - it suggests they both do the same thing - what they do is not clearly described. I have put an explanation of what they do in the linked issue above.
 - it says 'Singleline' is the default, but both options are off by default.

Possible alternative text might reference:

 - https://docs.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.regexoptions.multiline
 - https://docs.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.regexoptions.singleline

Version(s) of document impacted
------------------------------
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

NB. that any older versions of PowerShell will still forbid using them together, even if the documentation is updated and the linked issue is fixed in future releases.

### PR DESCRIPTION
**Pull request by bobbytreed**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.1 document
- [ ] Impacts 6.0 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Core/About/about_Split.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1631/reference/3.0/Microsoft.PowerShell.Core/About/about_Split.md) | :warning:Warning | | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Split.md)
[reference/4.0/Microsoft.PowerShell.Core/About/about_Split.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1631/reference/4.0/Microsoft.PowerShell.Core/About/about_Split.md) | :warning:Warning | | [Details](#user-content-reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Split.md)
[reference/5.0/Microsoft.PowerShell.Core/About/about_Split.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1631/reference/5.0/Microsoft.PowerShell.Core/About/about_Split.md) | :warning:Warning | | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Split.md)
[reference/5.1/Microsoft.PowerShell.Core/About/about_Split.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1631/reference/5.1/Microsoft.PowerShell.Core/About/about_Split.md) | :warning:Warning | | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Split.md)
[reference/6/Microsoft.PowerShell.Core/About/about_Split.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1631/reference/6/Microsoft.PowerShell.Core/About/about_Split.md) | :warning:Warning | | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Split.md)

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Split.md"></a>
### [reference/3.0/Microsoft.PowerShell.Core/About/about_Split.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1631/reference/3.0/Microsoft.PowerShell.Core/About/about_Split.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Split.md"></a>
### [reference/4.0/Microsoft.PowerShell.Core/About/about_Split.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1631/reference/4.0/Microsoft.PowerShell.Core/About/about_Split.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Split.md"></a>
### [reference/5.0/Microsoft.PowerShell.Core/About/about_Split.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1631/reference/5.0/Microsoft.PowerShell.Core/About/about_Split.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Split.md"></a>
### [reference/5.1/Microsoft.PowerShell.Core/About/about_Split.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1631/reference/5.1/Microsoft.PowerShell.Core/About/about_Split.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Split.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Split.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1631/reference/6/Microsoft.PowerShell.Core/About/about_Split.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C19%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806191445331072-2598%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=LZwx8lwOEJPM3JFoTzCO8a1BUjcuBIWl0QfMptylym0%3D&st=2018-06-19T14%3A50%3A56Z&se=2018-07-20T14%3A55%3A56Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Core/About/about_Split.md [hunk 1]
```diff
@@ -204,14 +207,23 @@ The RegexMatch options are:
...
 - **ExplicitCapture**: Ignores non-named match groups so that only
   explicit capture groups are returned in the result list. Valid
   only with RegexMatch.
 
+> [!NOTE]
+> **Singleline** and **Multiline** cannot be used together with the options
+> parameter, and **SingleLine** is the default.
+> This was resolved in PowerShell 6.0
+>
+> **Singleline** and **Multieline** can be combined using a *Mode-Modifier*.
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Core/About/about_Split.md]: Strike this statement. This does not apply to this version.

File: reference/3.0/Microsoft.PowerShell.Core/About/about_Split.md [hunk 2]
```diff
@@ -204,14 +207,23 @@ The RegexMatch options are:
...
+- **Multiline**: Multiline mode forces `^` and `$` to match the beginning
+  end of every line instead of the beginning and end of the input string.
+- **Singleline**: Singleline mode treats the input string as a *SingleLine*.
+  It forces the `.` character to match every character (including newlines),
+  instead of matching every character EXCEPT the newline `\n`.
 - **ExplicitCapture**: Ignores non-named match groups so that only
   explicit capture groups are returned in the result list. Valid
   only with RegexMatch.
 
+> [!NOTE]
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Core/About/about_Split.md]: Change this to :

> [!NOTE] 
> **SingleLine** is the default behavior. **Singleline** and **Multiline**
> cannot be used together with the options parameter. This was resolved in 
> PowerShell 6.0.

File: reference/4.0/Microsoft.PowerShell.Core/About/about_Split.md [hunk 3]
```diff
@@ -188,31 +191,40 @@ The syntax for the Options parameter is:
...
+- **Multiline**: Multiline mode forces `^` and `$` to match the beginning
+  end of every line instead of the beginning and end of the input string.
+- **Singleline**: Singleline mode treats the input string as a *SingleLine*.
+  It forces the `.` character to match every character (including newlines),
+  instead of matching every character EXCEPT the newline `\n`.
+- **ExplicitCapture**: Ignores non-named match groups so that only
   explicit capture groups are returned in the result list. Valid
   only with RegexMatch.
 
+> [!NOTE]
```
Comment (sdwheeler) [inline reference/4.0/Microsoft.PowerShell.Core/About/about_Split.md]: See previous comment.

File: reference/4.0/Microsoft.PowerShell.Core/About/about_Split.md [hunk 4]
```diff
@@ -188,31 +191,40 @@ The syntax for the Options parameter is:
...
+> **Singleline** and **Multiline** cannot be used together with the options
+> parameter, and **SingleLine** is the default.
+> This was resolved in PowerShell 6.0
+>
+> **Singleline** and **Multieline** can be combined using a *Mode-Modifier*.
```
Comment (sdwheeler) [inline reference/4.0/Microsoft.PowerShell.Core/About/about_Split.md]: Strike this statement.

File: reference/5.0/Microsoft.PowerShell.Core/About/about_Split.md [hunk 5]
```diff
@@ -188,31 +191,40 @@ The syntax for the Options parameter is:
...
+  end of every line instead of the beginning and end of the input string.
+- **Singleline**: Singleline mode treats the input string as a *SingleLine*.
+  It forces the `.` character to match every character (including newlines),
+  instead of matching every character EXCEPT the newline `\n`.
+- **ExplicitCapture**: Ignores non-named match groups so that only
   explicit capture groups are returned in the result list. Valid
   only with RegexMatch.
 
+> [!NOTE]
+> **Singleline** and **Multiline** cannot be used together with the options
```
Comment (sdwheeler) [inline reference/5.0/Microsoft.PowerShell.Core/About/about_Split.md]: See previous comment.

File: reference/5.0/Microsoft.PowerShell.Core/About/about_Split.md [hunk 6]
```diff
@@ -188,31 +191,40 @@ The syntax for the Options parameter is:
...
+> parameter, and **SingleLine** is the default.
+> This was resolved in PowerShell 6.0
+>
+> **Singleline** and **Multieline** can be combined using a *Mode-Modifier*.
```
Comment (sdwheeler) [inline reference/5.0/Microsoft.PowerShell.Core/About/about_Split.md]: Strike this statement.

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Split.md [hunk 7]
```diff
@@ -188,31 +191,40 @@ The syntax for the Options parameter is:
...
+- **Multiline**: Multiline mode forces `^` and `$` to match the beginning
+  end of every line instead of the beginning and end of the input string.
+- **Singleline**: Singleline mode treats the input string as a *SingleLine*.
+  It forces the `.` character to match every character (including newlines),
+  instead of matching every character EXCEPT the newline `\n`.
+- **ExplicitCapture**: Ignores non-named match groups so that only
   explicit capture groups are returned in the result list. Valid
   only with RegexMatch.
 
+> [!NOTE]
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Split.md]: See previous comment

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Split.md [hunk 8]
```diff
@@ -188,31 +191,40 @@ The syntax for the Options parameter is:
...
+> **Singleline** and **Multiline** cannot be used together with the options
+> parameter, and **SingleLine** is the default.
+> This was resolved in PowerShell 6.0
+>
+> **Singleline** and **Multieline** can be combined using a *Mode-Modifier*.
+> You can read more about mode modifiers in [Regular Expression Options](/dotnet/standard/base-types/regular-expression-options)
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Split.md]: Strike this statement.

File: reference/6/Microsoft.PowerShell.Core/About/about_Split.md [hunk 9]
```diff
@@ -188,31 +191,40 @@ The syntax for the Options parameter is:
...
+  end of every line instead of the beginning and end of the input string.
+- **Singleline**: Singleline mode treats the input string as a *SingleLine*.
+  It forces the `.` character to match every character (including newlines),
+  instead of matching every character EXCEPT the newline `\n`.
+- **ExplicitCapture**: Ignores non-named match groups so that only
   explicit capture groups are returned in the result list. Valid
   only with RegexMatch.
 
+> [!NOTE]
+> **Singleline** and **Multiline** cannot be used together with the options
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Core/About/about_Split.md]: I thought this was fix in this version?

File: reference/6/Microsoft.PowerShell.Core/About/about_Split.md [hunk 10]
```diff
@@ -188,31 +191,40 @@ The syntax for the Options parameter is:
...
+> parameter, and **SingleLine** is the default.
+> This was resolved in PowerShell 6.0
+>
+> **Singleline** and **Multieline** can be combined using a *Mode-Modifier*.
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Core/About/about_Split.md]: If you can use both options is this statement needed?

File: reference/3.0/Microsoft.PowerShell.Core/About/about_Split.md (hunk 2 above)
Comment (bobbytreed) [reply reference/3.0/Microsoft.PowerShell.Core/About/about_Split.md]: Fixed this

File: reference/4.0/Microsoft.PowerShell.Core/About/about_Split.md (hunk 3 above)
Comment (bobbytreed) [reply reference/4.0/Microsoft.PowerShell.Core/About/about_Split.md]: fixed this

File: reference/4.0/Microsoft.PowerShell.Core/About/about_Split.md (hunk 4 above)
Comment (bobbytreed) [reply reference/4.0/Microsoft.PowerShell.Core/About/about_Split.md]: fixed this

File: reference/5.0/Microsoft.PowerShell.Core/About/about_Split.md (hunk 5 above)
Comment (bobbytreed) [reply reference/5.0/Microsoft.PowerShell.Core/About/about_Split.md]: fixed this

File: reference/5.0/Microsoft.PowerShell.Core/About/about_Split.md (hunk 6 above)
Comment (bobbytreed) [reply reference/5.0/Microsoft.PowerShell.Core/About/about_Split.md]: fixed this

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Split.md (hunk 7 above)
Comment (bobbytreed) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_Split.md]: fixed this

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Split.md (hunk 8 above)
Comment (bobbytreed) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_Split.md]: fixed this

File: reference/6/Microsoft.PowerShell.Core/About/about_Split.md (hunk 9 above)
Comment (bobbytreed) [reply reference/6/Microsoft.PowerShell.Core/About/about_Split.md]: fixed this

File: reference/6/Microsoft.PowerShell.Core/About/about_Split.md (hunk 10 above)
Comment (bobbytreed) [reply reference/6/Microsoft.PowerShell.Core/About/about_Split.md]: removed whole block.

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Core/About/about_Split.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1631/reference/3.0/Microsoft.PowerShell.Core/About/about_Split.md) | :warning:Warning | | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Split.md)
[reference/4.0/Microsoft.PowerShell.Core/About/about_Split.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1631/reference/4.0/Microsoft.PowerShell.Core/About/about_Split.md) | :warning:Warning | | [Details](#user-content-reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Split.md)
[reference/5.0/Microsoft.PowerShell.Core/About/about_Split.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1631/reference/5.0/Microsoft.PowerShell.Core/About/about_Split.md) | :warning:Warning | | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Split.md)
[reference/5.1/Microsoft.PowerShell.Core/About/about_Split.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1631/reference/5.1/Microsoft.PowerShell.Core/About/about_Split.md) | :warning:Warning | | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Split.md)
[reference/6/Microsoft.PowerShell.Core/About/about_Split.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1631/reference/6/Microsoft.PowerShell.Core/About/about_Split.md) | :warning:Warning | | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Split.md)

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Split.md"></a>
### [reference/3.0/Microsoft.PowerShell.Core/About/about_Split.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1631/reference/3.0/Microsoft.PowerShell.Core/About/about_Split.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Split.md"></a>
### [reference/4.0/Microsoft.PowerShell.Core/About/about_Split.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1631/reference/4.0/Microsoft.PowerShell.Core/About/about_Split.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Split.md"></a>
### [reference/5.0/Microsoft.PowerShell.Core/About/about_Split.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1631/reference/5.0/Microsoft.PowerShell.Core/About/about_Split.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Split.md"></a>
### [reference/5.1/Microsoft.PowerShell.Core/About/about_Split.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1631/reference/5.1/Microsoft.PowerShell.Core/About/about_Split.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Split.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Split.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbyreed-issue1631/reference/6/Microsoft.PowerShell.Core/About/about_Split.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C19%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806191647528730-2598%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=RYmFOYNmJ8WSzQRp1S0WwSkQUE2I9XvY42TUK%2FeWz5U%3D&st=2018-06-19T16%3A53%3A19Z&se=2018-07-20T16%3A58%3A19Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**